RecoLocalTracker/Phase2TrackerRecHits/src/Phase2StripCPEDummy.cc

```diff
@@ -0,0 +1,17 @@
+
+#include "RecoLocalTracker/Phase2TrackerRecHits/interface/Phase2StripCPEDummy.h"
+
+
+Phase2StripCPEDummy::LocalValues Phase2StripCPEDummy::localParameters(const Phase2TrackerCluster1D & cluster, const GeomDetUnit & det) const {
+  float strippitch  = 0.0090; // hardcoded dummy, a la 2S
```

Comment:
It is a placeholder while we bring in the geometry dependence in a next step. It really doesn't matter what the exact value is.